src/LiftType.hs

```diff
@@ -67,38 +68,42 @@ liftType =
         case tr of
             Con tyCon ->
                 mk tyCon
-            App trA trB ->
-                AppT (go trA) (go trB)
             Fun trA trB ->
                 ConT ''(->) `AppT` go trA `AppT` go trB
+            App trA trB ->
+                AppT (go trA) (go trB)
 
     mk :: TyCon -> Type
     mk tyCon =
         let
             tcName =
                 tyConName tyCon
+            trySymbol = case tcName of
+              '"' : cs -> Just $ LitT (StrTyLit (zipWith const cs (drop 1 cs)))
+              _ -> Nothing
```

Comment:
Code style uses 4 spaces - I can fix this after.

Reply:
You'll have to. I'm lousy at formatting.